privacy.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: privacy
+title: Privacy Notice
+---
+
+# Foreman Privacy Policy
+
+This document describes the data Foreman collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Foreman project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * Email address
+* Data that identifies you
+  * IP addresses
+  * Browser UserAgent strings
+  * Debug archives
+  * Answers to survey questions
+
+# How and why we use your data
+
+## The website
+
+The Foreman website is a static site which does not track, collect, store
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
+The webserver logs are primarily used to produce aggregated statistics about
+the community, such as popularity of different operating system packages,
+plugins, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+The website does embed YouTube videos, see [YouTube's privacy policy][YouTube]
+for details. These videos do not autoplay.
+
+## Codebase
+
+The Foreman community use [GitHub](https://github.com) to manage the code that
+comprises the Foreman project. Refer to [GitHub's privacy policy][GitHub] for
+details on how GitHub handles the data you posts there. We also use
+non-identifiable data from the codebase to provide aggregated statistics such
+as average time to merge, number of open pull requests, etc.
+
+* Legal basis for this data usage: Legitimate interest
+
+The Foreman community also run a bug tracker using the
+[Redmine](http://www.redmine.org) software. When you sign up for a Redmine
+account, you provide us with an email address and a name. We only use this
+email within Redmine to contact you about bugs you have reported or subscribed
+to. You have full control over what emails are sent from your Redmine account
+preferences. The bug tracker also uses cookies to store your session.
+
+We also use the bug tracker data to provide aggregated statistics such as
+number of open/closed bugs, average time to close a bug, etc.
+
+* Legal basis for this data usage: Legitimate interest, consent
+
+## Communication
+
+The Foreman project maintains two public IRC channels on
+[Freenode](https://freenode.net), refer to [Freenode's privacy
+policy][Freenode] for details on how they handle your data. In addition, our
+*public* channels are logged to [BotBot.me](https://botbot.me) who also have a
+[privacy policy][BotBot].
+
+The Foreman community uses a forum using the [Discourse](https://discourse.org)
```

Comment:
Should it be noted that this is self-hosted? The current text could be interpreted as hosted by discourse.org.